### DISCUSSION TITLE
Setting up Windows CI on GitHub Actions

### DISCUSSION THREAD

**Discussion by longnguyen2004** [sqlitebrowser/sqlitebrowser#3131] [General]
There's an action for [setting up VS](https://github.com/marketplace/actions/setup-msbuild) and another for [Qt](https://github.com/marketplace/actions/install-qt), so we might be able to setup a Windows CI

**Comment by chrisjlocke** [comment]
Their documentation is poor - "vs-version: '[16.4,16.5)'" - opening square bracket, but closing normal bracket.

**Reply by scottfurry** [reply]
That looks similar to mathematical set notation. IIRC, indication of a range from value A to value B but doesn't include value B. Or it's the reverse, A to B but don't include A. I would have to look that up to be sure.

**Reply by longnguyen2004** [reply]
That's A to B, not including B, so at least 16.4 but less than 16.5

**Comment by lucydodo** [comment]
Sorry it took so long, this issue was resolved today. For further information refer: https://github.com/sqlitebrowser/sqlitebrowser/discussions/3508
